weblab/config/settings/test.py

```diff
@@ -1,4 +1,5 @@
 from .base import *  # noqa
+from .deployed import DATABASES
```

Comment:
Don't commit this - it's liable to break the CI tests. It should only be included on Vagrant VM deployments, which should be done in the Ansible configuration instead.

Reply:
or I can just do
```
try:
    from .deployed import DATABASES  # noqa
except ImportError:
    pass
```
a bit crude but it does work

Reply:
added vagrant.py with
```
from .base import *  # noqa
from .deployed import DATABASES  # noqa
```
and added instructions